prep-html-links/contact-us/contact-us.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Contact Us</title>
+</head>
+<body>
+  <h1>Contact Us</h1>
+  <div> <a href = "file:///Users/jmatsuda/Downloads/prep-html-links/index.html"></a>Click here to go to the home page</div>
```

Comment:
This won't work on somebody else's computer. You'll want to use a relative path back to the index file.

```html
<a href="../index.html"> ... </a>
```

Reply:
https://user-images.githubusercontent.com/72685552/134109434-b919f6f9-51bc-400c-ad95-dbd5f384c564.mov



Thank you for the help!